app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -0,0 +1,63 @@
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
+    require_nested :RefreshParser
+    require_nested :RefreshWorker
+    require_nested :Refresher
+
+    include AuthenticationMixin
+
+    DEFAULT_PORT = 80
+    default_value_for :port, DEFAULT_PORT
+    has_many :middleware_servers, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :middleware_deployments, :foreign_key => :ems_id, :dependent => :destroy
+
+    def verify_credentials(auth_type = nil, _options = {})
+      auth_type || 'default'
+      connect(_options).list_feeds
```

Comment:
Many of the other verify_credentials methods in other classes handle exceptions here so that the UI doesn't blow up.  Also, I think verify_credentials is a class method, but I could be wrong.